client/src/services/requestService.js

```diff
@@ -15,6 +16,16 @@ axios.interceptors.request.use(
     error => Promise.reject(error)
 );
 
+axios.interceptors.response.use(
+    response => response,
+    error => {
+        if (error.message.includes('401')) {
+            authService.removeToken();
+            router.push({ name: 'SignIn' });
+        }
+    }
+);
```

Comment:
It is not what we discussed. It is not what we need for a few reasons:
- It is not following the FLUX pattern.
- There are some cases when we must not redirect a user, for example, when user is trying to authenticate with the wrong credentials.
- It is better to check the response status code, at least, instead of searching in the response body.